sim/src/main/cc/bridges/TracerVModule.cc

```diff
@@ -0,0 +1,367 @@
+// See LICENSE for license details.
+
+#include "bridges/peek_poke.h"
+#include "bridges/tracerv.h"
+#include "core/bridge_driver.h"
+#include "core/simif.h"
+#include "core/simulation.h"
+
+#include <algorithm>
+#include <iostream>
+#include <string_view>
+
+static std::vector<bool> get_contiguous(const unsigned bits,
```

Comment:
While `clang-tidy` doesn't complain about primitive const in function definitions, please don't const-qualify all primitive arguments. It is quite noisy.